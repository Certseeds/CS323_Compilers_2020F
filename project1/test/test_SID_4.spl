int main(){
    char a = 0
}