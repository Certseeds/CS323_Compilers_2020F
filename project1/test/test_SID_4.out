Error type B at Line 2: Missing semicolon ';'
